.github/XE_SR_ERRORCODES.xml

```diff
@@ -0,0 +1,880 @@
+<?xml version="1.0" ?>

```

Comment:
Wouldn't it be better to have a way to copy this over from the SM repo instead of keeping here a file that will most likely get out of sync with the original one?

Reply:
sure, it can be fetched using wget. Last update was on 2020. Is it important to keep it in sync?

Reply:
This copy is just for CI testing. Consider it a mock file just for this purpose.

Reply:
hmm, let me try to download the file to the /opt/... location within the CI, that way we don't have to modify the Makefile

Reply:
The xml file is not updated too often and at the moment the output generated via this test is not used for anything, but theoretically this might change in future. Unless it causes issues, I'd rather have it done the right way, if only to avoid giving the wrong impression.